src/app/missions/detail/detail.component.html

```diff
@@ -162,31 +162,31 @@ <h4>Chat Bubbles</h4>
   <dl id="chat-bubbles">
     <div *ngIf="getChatState(textsLocale, 1); let text">
       <dt><blockquote class="chat-bubble" luxTooltip="chat_state_1" [innerHTML]="text"></blockquote></dt>
-      <dd>Spoken by <lux-object [luxFetch]="mission.offer_objectID"></lux-object> when approached before the mission is offered.</dd>
+      <dd>Spoken by <a [luxFetchObject]="mission.offer_objectID"></a> when approached before the mission is offered.</dd>
```

Comment:
If this is a standard anchor now, could that be written like so?
```suggestion
      <dd>Spoken by <a [luxFetchObject]="mission.offer_objectID"/> when approached before the mission is offered.</dd>
```

Reply:
Angular doesn't seem to like it:
```
Error: app/missions/tasks/npc-task/npc-task.component.html:3:1 - error NG5002: Only void and foreign elements can be self closed "a"
3 <a [luxFetchObject]="task.target"/>.
  ~~
  ```